moped-database/migrations/1703025658864_insert_into_public_moped_tags/up.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE "public"."moped_tags" SET "name" = 'Arterial Management - Inspection' WHERE "slug" = 'arterial_management';
+INSERT INTO "public"."moped_tags"("is_deleted", "id", "name", "slug", "type") VALUES (false, 62, E'Arterial Management - Request', E'arterial_management_request', E'Other');
+INSERT INTO "public"."moped_tags"("is_deleted", "id", "name", "slug", "type") VALUES (false, 61, E'Street Impact Fee (SIF)', E'street_impact_fee', E'Other');
```

Comment:
because this table has an auto-sequence set up for the `id` column, you want to exclude that column from the insert.

when you manually set the ID value, the sequence will actually throw a duplicate key error the next time someone tries to insert a row. you can test this running this SQL after you apply these migrations:

```sql
INSERT INTO "public"."moped_tags"("is_deleted","name", "slug", "type") VALUES (false, E'Blah blah', E'blah', E'blah');
```

> `Query 1 ERROR at Line 1: : ERROR:  duplicate key value violates unique constraint "moped_tags_pkey"
DETAIL:  Key (id)=(61) already exists.`

Reply:
thanks for pointing this out! migrations updated :)